### ISSUE TITLE
Best terms for Webfinger -> Actor ID and Actor ID -> Webfinger

### ISSUE THREAD

**Issue by evanp** [swicg/activitypub-webfinger#12]
Currently the doc uses "Discovery" for the process of getting an Actor ID from a Webfinger address.

It uses "Encoding" for the process of getting a Webfinger address given an Actor ID.

I changed this from "forward discovery" and "reverse discovery" since those were a little hard to parse.

Regardless of the terms, I think they should each be top-level headings.

**Comment by trwnh** [comment]
Oh, I was using Encoding to talk about allowed characters, MIME types, `links` formatting, and stuff like that

**Comment by trwnh** [comment]
After thinking about this, I think it's actually better to have them under the "Discovery" header, since they both deal with discovering some identifier given another identifier. I also don't see a problem with "forward" or "reverse" as disambiguation. I have more of a problem with "Encoding" being used to describe "reverse discovery", because to me, "encoding" is the best way to describe the section that's actually about encoding the identifiers and the responses.

**Comment by trwnh** [comment]
Especially noteworthy is that issue #4 was filed as "reverse discovery", so I'm just taking that terminology :)
